app/controllers/katello/api/v2/subscriptions_controller.rb

```diff
@@ -17,8 +17,6 @@ class Api::V2::SubscriptionsController < Api::V2::ApiController
     end
 
     api :GET, "/organizations/:organization_id/subscriptions", N_("List organization subscriptions")
-    api :GET, "/systems/:system_id/subscriptions", N_("List a content host's subscriptions"), :deprecated => true
-    api :GET, "/activation_keys/:activation_key_id/subscriptions", N_("List an activation key's subscriptions")
```

Comment:
Showing subscriptions for systems and activation keys require calling candlepin directly and returning what it sends in the API so we can have the quantity attached to a system or activation key. This wouldn't' return Pool (or Subscription) objects from the database but rather a presenter (SystemSubscriptionPresenter  or ActivationKeySubscriptionPresenter) and the subscription index call would be returning 2 different types of objects, which is further complicated by the index call using scoped search. I felt it would be simpler to move the calls to the system and activation key controllers.